libraries/tock-cells/src/map_cell.rs

```diff
@@ -2,7 +2,66 @@
 
 use core::cell::{Cell, UnsafeCell};
 use core::mem::MaybeUninit;
+use core::ops::{Deref, DerefMut};
 use core::ptr;
+use core::ptr::drop_in_place;
+use map_cell::MapCellErr::AlreadyBorrowed;
```

Comment:
nit: it'd be more readable to use `self` here and on the line below

Reply:
Sure!